docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
         restart: "on-failure:10"
 
     postgres:
-        image: codefordc2/housing-insights-postgres
+        image: mdillon/postgis
```

Comment:
What was your motivation for changing this postgres image? I was able to get it running on my machine when I switched back to the existing postgres, though I only tested the crime endpoint. the `mdillon/postgis` image uses Postgres 9.6, while we're still on 9.5 - switching versions is ok if it's needed b/c AWS [now supports 9.6](https://aws.amazon.com/about-aws/whats-new/2016/11/amazon-rds-now-supports-postgresql-9-6-1/) but it will cause problems with anyone that is using their local Docker to run the database - they'll have to delete the associated volume b/c the existing data clashes.  If we can keep the existing image that would be easiest.